### ISSUE TITLE
Add geo_list class to output from geojson_list.list?

### ISSUE THREAD

**Issue by ateucher** [ropensci/geojsonio#67]
Should we wrap the return value from [here](https://github.com/ropensci/geojsonio/blob/master/R/geojson_list.R#L303) in `as.geo_list`?

```r
geojson_list.json <- function(input, lat = NULL, lon = NULL, group = NULL,
                              geometry = "point", type = "FeatureCollection", ...) {

  output_list <- jsonlite::fromJSON(input, FALSE, ...)
  as.geo_list(output_list, from = "list)
}
```

**Comment by sckott** [comment]
Yeah, I think so. PR? or I can do

**Comment by ateucher** [comment]
PR sent. Should work, though I did it straight in GitHub, so didn't test it.

**Comment by ateucher** [comment]
Closed via #68 and 6ec75aa709
